tools/bazel.rc

```diff
@@ -1,6 +1,8 @@
 build:cuda --crosstool_top=@org_tensorflow//third_party/gpus/crosstool
 build:cuda --define=using_cuda=true --define=using_cuda_nvcc=true
 
+build:mkl --define=using_mkl=true
+
```

Comment:
It would also make sense to do the same thing as TF does here: https://github.com/tensorflow/tensorflow/blob/master/tools/bazel.rc